bin/ch/Helpers.cpp

```diff
@@ -4,14 +4,13 @@
 //-------------------------------------------------------------------------------------------------------
 #include "stdafx.h"
 
-HRESULT Helpers::LoadScriptFromFile(LPCSTR filename, LPCSTR& contents, bool* isUtf8Out /*= nullptr*/, UINT* lengthBytesOut /*= nullptr*/)
+HRESULT Helpers::LoadScriptFromFile(LPCSTR filename, LPCSTR& contents, UINT* lengthBytesOut /*= nullptr*/)
 {
     HRESULT hr = S_OK;
     BYTE * pRawBytes = nullptr;
     UINT lengthBytes = 0;
-    bool isUtf8 = false;
     contents = nullptr;
-    FILE * file;
+    FILE * file = nullptr;
```

Comment:
Reminder: Although nullptr and NULL are equivalent by the definition..`fopen` is from stdio.h and returns NULL in case of failure (c++11 standard doesn't have a guarantee for .h files). Besides, fopen_s is from PAL and assigns NULL to FILE.